chirp/src/db/content.rs

```diff
@@ -198,13 +203,114 @@ pub fn db_content_add(contents: Vec<FullContent>) -> Result<(), Box<dyn Error +
     Ok(())
 }
 
+pub fn db_content_add_words_phrases(
+    cc_id: i32,
+    cb: ContentBody,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let clean_text = strip_html_tags_from_string(&cb.body_text);
+    let phrases_tallies = collect_word_tallies_with_intersections(&clean_text);
+
+    let mut content_ids: Vec<i32> = Vec::new();
+    let mut phrases: Vec<String> = Vec::new();
+    let mut tallies: Vec<i32> = Vec::new();
+
+    for (phrase, tally) in phrases_tallies {
+        // println!("c {:1}, p {:2}, t {:3}", &cc_id, &phrase.join(" "), &tally);
+        content_ids.push(cc_id);
+        phrases.push(phrase.join(" "));
+        tallies.push(tally);
+    }
+
+    let c_ids_r = create_rarray_values(content_ids);
```

Comment:
I think you can possibly reference the one Content ID with the `params!` macro, that may not have the same equivalent type constraints as a normal array? Or maybe not?

Reply:
Meaning because there is only one content id you can just pass that into `params!` without duplicating it to same number of rows.

Reply:
ok that worked pretty well and it's probably faster now too

Reply:
great work, but it seems the argument order in the params could be improved.